libcodechecker/libhandlers/parse.py

```diff
@@ -72,6 +73,29 @@ def add_arguments_to_parser(parser):
                         help="Specify the format the analysis results were "
                              "created as.")
 
+    output_opts = parser.add_argument_group("output arguments")
+    output_opts.add_argument('--output-format',
+                             dest="output_format",
+                             required=False,
+                             choices=['html'],
```

Comment:
Introducing this argument here will make `parse` something like the `plist` command was, certain arguments completely derailing operation into a different mode, which we spent a lot of time to get rid of.

I think it's better if `plaintext` is also available for the `--output-format` option. And if it is specified, create the "usual" output. If `html` is given, create HTML files. Mixing the two feels really weird, it should be two distinct invocations

**or** the opt-string should not be called `output-format`, as clearly it does not **switch** the output format, but orders the command to do an **extra export step**.

Reply:
I renamed the opt-string to `export`